kong/pdk/log.lua

```diff
@@ -791,72 +804,56 @@ do
           upstream_uri = upstream_uri .. "?" .. (var.args or "")
         end
       end
-
-      return edit_result(ctx, {
+      local root = {
```

Comment:
`root` is not a good name.

Reply:
`root` means root node, it's named after the param of `local function edit_result(ctx, root)`

Reply:
This `root` var is necessary? Is the original form not OK?

Reply:
any better name is welcomed

Reply:
> Is this `root` var necessary? Is the original form not OK?

we can merge the same part into the different part later, reduce duplicated code, which requires the identifier name

Reply:
I saw the function, I think it is OK.